sdk/purview/Azure.Analytics.Purview.Workflows/CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Release History
+
+## 1.0.0-beta.2 (Unreleased)
+
+### Features Added
+
+### Breaking Changes
+
+### Bugs Fixed
+
+### Other Changes
+
+## 1.0.0-beta.1
```

Comment:
Can you point me to where this version was released in NuGet?   It looks like we have a single changelog entry for `1.0.0-beta.1` and remove `beta.2`

Reply:
Fixed, thanks